packages/nuxt/src/core/templates.ts

```diff
@@ -334,6 +334,21 @@ export const publicPathTemplate: NuxtTemplate = {
   }
 }
 
+export const dollarFetchTemplate: NuxtTemplate = {
+  filename: 'fetch.mjs',
+  getContents () {
+    return [
+      "import { $fetch } from 'ofetch'",
+      "import { baseURL } from '#build/paths.mjs'",
```

Comment:
Would it make sense/be possible to move these inside the if statement and import dynamically?